HWweek6.py

```diff
@@ -0,0 +1,87 @@
+class TreeNode(object):
```

Comment:
Your code is very clean/concise :)

That makes this a bit of a nitpick, but don't forget to leave comments in your pull requests. It helps me see your thought-process more clearly, and therefore works as a good way to practice the "Talk" step.